requirements.txt

```diff
@@ -7,5 +7,6 @@ requests
 rdflib
 rdflib-jsonld
 mf2py>=1.1.0
-six
+six>=1.11
```

Comment:
@jakubwasikowski why did you updated six version? Maybe this is useful for @croqaz in #120

Reply:
Yes, I did exactly the same thing, but updated to 1.12 :D

Reply:
Sorry, forgot to mention it in description of PR. I had to select minimal version because six in version 1.10.0 causes errors for python3.4 when installing.

Reply:
So when I added six>=1.12 it still crashes for Python 3.4.
But six>=1.11 works 🤦‍♂

Reply:
For 1.11 it works, but for 1.12 it doesn't? Weird :man_facepalming: Especially taking into account that 1.12 is the latest version.